mPower2/mPower2/View Controllers/TaskBrowserViewController.swift

```diff
@@ -299,8 +308,15 @@ extension TaskBrowserViewController: UICollectionViewDelegate, UICollectionViewD
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: collectionCellIdentifier, for: indexPath) as? TaskCollectionViewCell
         let scheduledTask = tasks[indexPath.row]
         let task = scheduledTask.taskInfo
-        cell?.image = task.iconWhite
-        cell?.title = task.title?.uppercased()
+        if (scheduledTask.identifier == RSDIdentifier.heartSnapshotTask.identifier) {
```

Comment:
Wish this just happened automatically, but CRFTaskInfo didn't have the "iconWhite" variable so had to use a conditional check.

Reply:
```
extension RSDTaskInfo {
    var iconWhite: UIImage? {
        get {
            return UIImage(named: "\(self.identifier)TaskIconWhite")
        }
    }
}
```

You need to add it to the asset catalog. *Everything* about mPower is custom. Since this doesn't fit the model that we'd originally come up with... oh well.